src/tiledimage.js

```diff
@@ -1771,7 +1774,7 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
 
         });
 
-        return tiles;
+        return tiles || [];
```

Comment:
In tests this sometimes threw an error as it returned null, I was unable to isolate the cause, but still - if a method says it returns an array, it should not return null anyway.